cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -192,11 +192,11 @@ private PublicationDate extractPublicationDate() {
     }
 
     private String extractMainTitle() {
-        return extractCristinTitles()
-                   .filter(CristinTitle::isMainTitle)
-                   .findFirst()
-                   .map(CristinTitle::getTitle)
-                   .orElseThrow();
+        var x = extractCristinTitles()
```

Comment:
I would make a remark, but x marks the spot

Reply:
woops :1st_place_medal: